anchore_engine/services/apiext/swagger/swagger.yaml

```diff
@@ -429,6 +429,37 @@ paths:
           description: Internal Error
           schema:
             $ref: "#/definitions/ApiErrorResponse"
+    delete:
+      tags:
+      - Images
+      x-swagger-router-controller: anchore_engine.services.apiext.api.controllers.images
+      operationId: delete_images_async
+      x-anchore-authz-action: deleteImage
+      summary: Bulk mark images for deletion
+      description: Delete analysis for image digests in the list asynchronously
+      parameters:
+      - name: imageDigests
+        in: query
+        required: true
+        x-nullable: false
+        type: array
+        items:
+          type: string
+        collectionFormat: csv
```

Comment:
We have support for array params in the /query/vulnerabilities route as well. It uses the default collection format which is csv. I think this is good, should be explicit. I can update the other value to be explicit as well.